clarity.d.ts

```diff
@@ -58,6 +58,7 @@ interface IEnvelope {
   clarityId: string;
   impressionId: string;
   url: string;
+  title: string;
```

Comment:
Sending this in every envelope seems consistent with the current design but a not great design. Maybe it's time to change this and create an explicit event with "page data" that includes this and send it and other common information only once rather than as part of every envelope? Also do we not pick up changes to the <title> element using normal DOM mutations?